place/views.py

```
@@ -0,0 +1,68 @@
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+        
+        lat = serializer.validated_data["latitude"]
+        lng = serializer.validated_data["longitude"]
+        radius = serializer.validated_data.get("radius", 3000)
+        keyword = serializer.validated_data.get("keyword", "museum")
+
+        # 🔽 여기가 핵심: 비동기 함수를 동기에서 호출할 수 있도록 `async_to_sync` 사용
+        from asgiref.sync import async_to_sync
+        try:
+            payload = {
```

Ruff (0.12.2)

Local variable `payload` is assigned to but never used

Remove assignment to unused variable `payload`

(F841)